Microsoft.Toolkit.Uwp.SampleApp/SamplePages/MasterDetailsView/MasterDetailsView.bind

```diff
@@ -0,0 +1,36 @@
+﻿<Page
```

Comment:
The file format is not using the right encoding and thus it is not displayed correctly in the sample page (should be 1252)